hypervisor/arch/x86/trusty.c

```diff
@@ -461,6 +468,14 @@ bool initialize_trusty(struct vcpu *vcpu, uint64_t param)
 	/* save Normal World context */
 	save_world_ctx(&vcpu->arch_vcpu.contexts[NORMAL_WORLD]);
 
+	if (boot_param.version == TRUSTY_VERSION_2) {
+		/* copy rpmb_key from OSloader */
+		memcpy_s(&g_key_info.rpmb_key[0][0], 64,
+				&boot_param.rpmb_key[0], 64);
+		memset(&boot_param.rpmb_key[0], 0, 64);
```

Comment:
2nd / 3th parameter of memset is uint8/ size_t, which is unsigned integer. Please use 0U, 64U...  This is MISRA-C requirement, and we need to enforce now.

Reply:
@dongyaozu Has fixed in latest version.